packages/core/src/components/callout/_callout.scss

```diff
@@ -45,7 +45,7 @@ Styleguide callout
   &.#{$ns}-callout-icon {
     padding-left: ($pt-grid-size * 2) + $pt-icon-size-large;
 
-    .#{$ns}-icon {
+    > .#{$ns}-icon {
```

Comment:
might actually want to add `:first-child` here to avoid targeting potential icons in the callout text.